Gemfile.lock

```diff
@@ -275,6 +275,14 @@ GEM
     nokogiri (1.13.4)
       mini_portile2 (~> 2.8.0)
       racc (~> 1.4)
+    opensearch-api (1.0.0)
+      multi_json
+    opensearch-ruby (1.0.0)
+      opensearch-api (= 1.0.0)
+      opensearch-transport (= 1.0.0)
```

Comment:
We will have both opensearch-api, opensearch-transport and elasticsearch-api,elasticsearch-transport. Have you looked into which one is really being used? How are elasticsearch-* modules requiring opensearch-*? has opensearch overridden same classes?